### DISCUSSION TITLE
Interface to type in date

### DISCUSSION THREAD

**Discussion by Dumpstar56** [directus/directus#14831] [Feature Requests]
Our database project deals with history, so all of our dates are far in the past. The current date picker is very cumbersome to navigate far into the past. It would be great to have an interface option for dates where I can just type in the date rather than use the calendar date picker. Our administrator created the option below, which has the date picker and I can just type in - but it has started to throw errors and a built-in solution would be great!
![type data](https://user-images.githubusercontent.com/94397183/182657328-8bdabda1-e795-43a5-9771-9ba963c574d1.jpg)

**Comment by benhaynes** [comment]
While I agree that a "type-able" datetime interface mask would be ideal, I wonder if setting a distant default time for the field would at least get the initial value far enough in the past to make selecting a date easier as a stop-gap.

**Reply by rijkvanzanten** [reply]
> I wonder if setting a distant default time for the field would at least get the initial value far enough in the past to make selecting a date easier as a stop-gap.
 
I guess that would cause the same problem "the other way" though wouldn't it? Then it would become extra cumbersome to select a present/future date 🤔

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
